chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1267,6 +1267,184 @@
 						"description": "Expect Listing Basic response to have svapResultsUrl"
 					},
 					"response": []
+				},
+				{
+					"name": "PUT /{certifiedProductId}- 400 status and error on adding removed/invalid Test Tools to 2015 listings",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"var ReqBody= {\r",
+									"        userName:pm.environment.get(\"usernameONC\"),password:pm.environment.get(\"passwordONC\")\r",
+									"    };\r",
+									"pm.sendRequest({\r",
+									"    url: pm.environment.get(\"authurl\"),\r",
+									"    method: \"POST\",\r",
+									"    header: {\r",
+									"        \"Content-Type\": \"application/json\",\r",
+									"        \"API-KEY\": pm.environment.get(\"apiKey\")\r",
+									"    },\r",
+									"    body: JSON.stringify(ReqBody)\r",
+									"}, function (err, response) {\r",
+									"   var jsonData = response.json();\r",
+									"   pm.environment.set(\"token\",jsonData[\"token\"])\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Adding removed Test Tools to 2015 listing should give 400 status code and error in response\", function () {\r",
+									"    pm.response.to.have.status(400);\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'Transport Testing Tool' is not valid\");\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'Electronic Prescribing' is not valid\");\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'HL7 CDA National Health Care Surveys Validator' is not valid\");\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'HL7 v2 Immunization Information System (IIS) Reporting Validation Tool' is not valid\");\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'HL7 v2 Laboratory Results Interface (LRI) Validation Tool' is not valid\");\r",
+									"    pm.expect(pm.response.text()).to.include(\"Test Tool 'HL7 v2 Syndromic Surveillance Reporting Validation Tool' is not valid\");\r",
+									"});\r",
+									"pm.test(\"Adding invalid Test Tool to listing should give 400 status code and error in response\", function () {\r",
+									"    pm.response.to.have.status(400);\r",
+									"    pm.expect(pm.response.text()).to.include(\"contains an invalid test tool 'HL7 CDA Cancer Registry Reporting Validation Tool'\");\r",
```

Comment:
What is different between these 2 tests? It reads same to me. line #1301 and 1310.
THL7 CDA Cancer Registry Reporting Validation Tool is actually valid test tool for 2015 listings for f4 criteria

Reply:
can you update test case name to reflect that please?
Same goes for 2014 listings

Reply:
Done